determine_test_coverage.py

```diff
@@ -65,8 +67,10 @@
         test_name, backend = test_backend.split(",")
         print("Test:", test_backend)
         command = (
-            f'docker run -v "$(pwd)":/ivy unifyai/ivy:latest timeout 30m /bin/bash -c "coverage run --source=ivy,'  # noqa
-            f"ivy_tests -m pytest {test_name} --backend {backend} --disable-warnings > coverage_output;coverage "  # noqa
+            f'docker run -v "$(pwd)":/ivy unifyai/ivy:latest timeout 30m \
+            /bin/bash -c "coverage run --source=ivy,'
+            f"ivy_tests -m pytest {test_name} --backend {backend} --disable-warnings \
+            > coverage_output;coverage " 
```

Comment:
As these are system commands they shouldn't be tampered with, please revert that change